bundles/org.openhab.voice.googlestt/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.voice.googlestt</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Google Cloud Speech to Text</name>
+  <properties>
+    <bnd.importpackage>!*opencensus*,!org.bouncycastle*,!*jboss*,!javax.annotation.*,!net.jpountz.*,!lzma.sdk.*,org.eclipse.jetty.*;resolution:=optional,com.ning.*;resolution:=optional,com.jcraft.*;resolution:=optional,com.google.re2j.*;resolution:=optional,com.google.api.client.*;resolution:=optional,org.conscrypt.*;resolution:=optional,!io.grpc.census.*,com.sun.jndi.dns.*;resolution:=optional,org.apache.log.*;resolution:=optional,org.apache.http.*;resolution:=optional,sun.security.*;resolution:=optional,com.oracle.svm.core.annotate.*;resolution:=optional,*blockhound*;resolution:=optional,com.google.protobuf.nano.*;resolution:=optional,io.grpc.*;resolution:=optional,com.google.protobuf.*;resolution:=optional,io.perfmark.*;resolution:=optional</bnd.importpackage>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-speech</artifactId>
+      <version>2.2.2</version>
+    </dependency>
+    <!--cloud-speech deps -->
+    <dependency>
```

Comment:
To be checked by someone more expert than me, including if someone has to be added in your feature.xml file.

Reply:
Yes please, I'm also not an expert on this matter. This is the smallest set of dependencies I've been able use to made it work.

Reply:
Like in your other binding (spotter), I think it will better to add the "compile".

Reply:
@wborn : is it better to add compile tag also on all dependencies of the main dependency ?

Reply:
It's subjective if it is better or not to add "compile" as scope because it is the default scope anyways. With openHAB I usually add it because it is more common to do that here but on other projects I don't.